### PR TITLE
[test-only] ApiTest. Check email notification ocminvite

### PR DESCRIPTION
**Pull request by ScharfViktor**
related https://github.com/owncloud/ocis/issues/9583

test for receiving an email if the user specified an email when creating a federation share invitation

### CODE REVIEW

File: tests/acceptance/bootstrap/OcmContext.php [hunk 1]
```diff
@@ -68,6 +68,13 @@ public function getFedOcisDomain(): string {
 		return $this->extractDomain(\getenv('TEST_SERVER_FED_URL'));
 	}
 
+	/**
+	 * @return string
+	 */
+	public function getLastInvitationToken():string {
```
Comment (saw-jan) [inline tests/acceptance/bootstrap/OcmContext.php:74]:
```suggestion
	public function getLastFederatedInvitationToken():string {
```

File: go.mod
Comment (saw-jan) [inline go.mod:1]: are these changes required?

**Review by saw-jan — COMMENTED** [review]
others look good

File: go.mod
Comment (ScharfViktor) [reply go.mod:1]: No, I'll delete it now.

**Review by saw-jan — APPROVED** [review]
:+1:
